explainability-service/manifests/opendatahub/default/kustomization.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ./trusty-deployment.yaml
```

To be renamed as 
```suggestion
  - ./trustyai-deployment.yaml
```
together with the file